### ISSUE TITLE
Cryptic error when Comsol version is missing

### ISSUE THREAD

**Issue by max3-2** [MPh-py/MPh#42]
John,

I stumbled across the following when starting the server with a wrong version, e.g. a typo or just a version not installed or not found, e.g `server = Server(version=5.5.)` will produce

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/mph/server.py", line 74, in __init__
    backend = discovery.backend(version)
  File "/usr/local/lib/python3.9/site-packages/mph/discovery.py", line 419, in backend
    return backends[names.index(version)]
ValueError: '5.4' is not in list
```

which I think is a bit cryptic. I suggest adding an e.g. `logger.error('The specified version was not found)`

**Comment by John-Hennig** [comment]
Yeah, you're right, that's not pretty. I'll fix that in 1.0.4. It should keep raising an error, but have a clearer message. Thanks for reporting.

**Comment by John-Hennig** [comment]
Fixed in version 1.0.4, released today.
